src/aktualizr.cc

```diff
@@ -68,8 +68,12 @@ int Aktualizr::run() {
 #endif
   } else {
 #ifdef BUILD_OSTREE
-    // TODO: compile unconditionally
+// TODO: compile unconditionally
+#ifdef BUILD_P11
     boost::shared_ptr<INvStorage> storage = INvStorage::newStorage(config_.storage);
+#else
+    boost::shared_ptr<INvStorage> storage = INvStorage::newStorage(config_.storage);
+#endif
```

Comment:
Aren't both legs of this if the same?

Reply:
Oh, yes, I made wrong merge conflict resolve